### PR TITLE
doc: final tutorial polishing

### PR DESCRIPTION
**Pull request by Rqnsom**
- Improved intro sections.
- Improved final "finalizing the tutorial" section.
- Put images in tables so it looks nicer.

### CODE REVIEW

**Comment by Rqnsom** [comment]
Clippy is failing because of https://github.com/tkaitchuck/aHash/issues/200

File: doc/tutorial.md [hunk 1]
```diff
@@ -144,37 +153,74 @@ Estimated gas: Estimate (gas_used: 21, vm_status_code: EXECUTED)
...
-  - Hint 1: because the price of a washing coin is 1 UNIT), we need to write a cheque of around xxxx
- 
-3. Wash the car with Alice's account 
-  - Hint 1: ...
-
-After the car is washed, now check Alice's balance again and see how the above operations changed her balance status:
-![Compare balances after Alice bought a coin from Bob](assets/polkadot.js_buy_coin_balances_after.png)
-
-We hope you finished our tutorial without any issues! And we hope you liked it. If you want to report feedback, please feel free to do so. We are always available at [hello@eiger.co](mailto:hello@eiger.co).
+  - _Hint: Because the price of a washing coin is `1 UNIT`, we need to write a cheque with a value of at least `1000000000000` balances._
```
Comment (neutrinoks) [inline doc/tutorial.md:190]: Mhh, what about deleting the last word `balances`? Looks a little strange to me here

**Review by neutrinoks — APPROVED** [review]
Two comments for minor details. You can decide if you want to change that :+1:

But very cool innovation:
- Short URLs in text
- Linebreaks for each sentence, since Markdown still reforms it to a paragraph
- Hidden hints
- Borders and captions on images

Very cool :+1: :+1: :+1:

**Review by neutrinoks — DISMISSED** [review]
Looks good :+1:

File: doc/tutorial.md [hunk 2]
```diff
@@ -16,19 +16,26 @@
...
+Finally, we publish the module and execute a script via using [polkadot.js][polkadotjs]).
 
 
 ## Setup and Code Example 
 
-Initial requirements:
+Prerequisites:
 - [Install smove](https://github.com/eigerco/smove).
-- Setup our [our template node](https://github.com/eigerco/substrate-node-template-move-vm-test) with pallet-move integrated and run it in the background.
+- Setup the our template node with pallet-move integrated and run it in the background - the instructions can be found in our [readme](../README.md).
```
Comment (asmie) [inline doc/tutorial.md:32]:
```suggestion
- Setup our template node with pallet-move integrated and run it in the background - the instructions can be found in our [readme](../README.md).
```

File: doc/tutorial.md [hunk 3]
```diff
@@ -16,19 +16,26 @@
 
 ## Overview
 
+In this tutorial, we are going to show you how to access the MoveVM hosted inside the `pallet-move`, integrated inside the [template node][template-node-repo].
+
 To make Move interoperable with Substrate, we provide the package manager [smove](https://github.com/eigerco/smove), which can compile your Move projects, create bundles, access and act with the network node and more.
 
-This tutorial shows a summary of a workflow with Move on a Substrate template node. We will publish a module and execute a script which uses that module's functionality. Therefore, the package manager `smove` will be used to compile those resources, estimate the required amount of gas, and to create the `script-transaction`. Finally, we publish the module and execute a script via using [polkadot.js](https://polkadot.js.org/apps/).
+This tutorial shows a summary of a workflow with Move on a Substrate template node. We will publish a module and execute a script which uses that module's functionality.
+Therefore, the package manager `smove` will be used to compile those resources, estimate the required amount of gas, and to create the `script-transaction`.
```
Comment (asmie) [inline doc/tutorial.md:24]:
```suggestion
Therefore, the package manager `smove` will be used to compile those resources, estimate the required amount of gas, and create the `script-transaction`.
```

**Review by asmie — APPROVED** [review]
LGTM!

**Review by asmie — APPROVED** [review]
:+1:

**Pull request merged**